jerry-core/config.h

```diff
@@ -180,4 +180,12 @@
  */
 #define CONFIG_PARSER_ENABLE_PARSE_TIME_BYTE_CODE_OPTIMIZER
 
+/**
+  * Enable timezone and daylightsaving system calls for date object
+  *
+  * Note:
+  *      Experimental. Works on Ubuntu 14.04.
+  */
+// #define ECMA_ENABLE_DATE_SYS_CALLS
```

Comment:
Is there a way to enable the calls without editing config.h? If not, we wont be testing it "ever" -- neither with local precommit checks, nor on Travis. Which makes all gettimeofday-related code paths effectively dead. External tool chains are not the primary targets (strictly IMHO). If their users need tweaks, they shoud be the one to do edits to config.h.